airbyte-integrations/connector-templates/source-java-jdbc/acceptance-test-docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
```

Comment:
Please reuse https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source_acceptance_test_files/acceptance-test-config.yml.hbs#L1

Reply:
let discuss it here https://github.com/airbytehq/airbyte/pull/19660/files/eea4d4b555919f46d790572f3a1359450e930337#r1028406403

Reply:
Sorry I meant this file https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source_acceptance_test_files/acceptance-test-docker.sh#L1

Your new file is not different from this one isn't it?

Reply:
:+1: